string data

float32 center_x
float32 center_y
float32 angle
float32 width
float32 height

# Point order is left/top, left/bottom, right/bottom, right/top
int32[4] ptx
int32[4] pty
